hadoop-ozone/tools/src/main/java/org/apache/hadoop/ozone/admin/om/OMAdmin.java

```diff
@@ -50,7 +51,7 @@
     subcommands = {
         GetServiceRolesSubcommand.class
     })
-public class OMAdmin extends GenericCli {
+public class OMAdmin extends GenericCli implements SubcommandWithParent {
```

Comment:
Won't we need

```
@MetaInfServices(SubcommandWithParent.class)
```

too?

Currently `OMAdmin` is added to `OzoneAdmin` from the parent, so it's not a problem.  (#1285 will allow inverting subcommand registration for `OzoneAdmin`, too.)

Reply:
Yes, it's missing. :eagle: :eye: :trophy: 

Also, I can remove the `OmAdmin` from the annotations of `OzoneAdmin`. (Fortunately annotations and subcommands added programmatically can work together).